model/historical_test_data.go

```diff
@@ -0,0 +1,69 @@
+package model
+
+import (
+	"time"
+
+	"github.com/evergreen-ci/cedar"
+	"github.com/pkg/errors"
+)
+
+// HistoricalTestData describes aggregated test result data for a given date
+// range.
+type HistoricalTestData struct {
+	Info            HistoricalTestDataInfo
+	NumPass         int       `bson:"num_pass"`
+	NumFail         int       `bson:"num_fail"`
+	AverageDuration float64   `bson:"average_duration"`
+	LastUpdate      time.Time `bson:"last_update"`
+
+	env       cedar.Environment
+	populated bool
+}
+
+// CreateHistoricalTestData is an entry point for creating a new
+// HistoricalTestData.
+func CreateHistoricalTestData(info HistoricalTestDataInfo) (*HistoricalTestData, error) {
+	if err := info.validate(); err != nil {
```

Comment:
why is this private?

Reply:
it doesn't have to be but I didn't see the need to make it public?